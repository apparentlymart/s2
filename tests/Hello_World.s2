#-*-c++-*-

layerinfo "type" = "core";

property string blah;
set "blah" = "foo";

function main ()
{
  "hi.\n";
  print "hi.\n";
  println "hi.";
  """triple quote print\n""";
  print """triple quote print2\n""";
  """<div id="comments">\n""";
  """<div class="topiclinks">\n""";
}



