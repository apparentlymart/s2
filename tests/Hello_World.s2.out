hi.
hi.
hi.
triple quote print
triple quote print2
<div id="comments">
<div class="topiclinks">
